web/common/routes.js

```diff
@@ -72,9 +72,32 @@ export const ROUTES = [
   {
     path: "/home",
     label: "Mes informations",
-    accessible: ({ userInfo }) => !!userInfo.id,
+    accessible: () => true,
+    menuItemFilter: () => false,
     component: Home
   },
+  {
+    path: "/logout",
+    label: "Déconnexion",
+    accessible: () => true,
+    menuItemFilter: () => false,
+    component: Logout
+  },
+  {
```

Comment:
Il faut mettre ça tout à la fin de la liste. En effet les routes sont rendered par React en respectant l'ordre de la liste => le matching d'un URL avec une route se fait aussi dans l'ordre de cette liste  . 

Le problème est ici que le `path` de la route est "". Or React vérifie qu'un URL matche une route en utilisant une Regexp plutôt qu'un match exact (voir https://reactrouter.com/web/api/Redirect/to-string). Ici tout URL va matcher cette route => combiné avec le premier comportement, toutes les routes qui sont définies après celle-ci dans la liste sont dans le faits inaccessibles.

Désolé c'est un comportement un peu caché